### PR TITLE
Add diagrams for GitHub Integration

### PR DESCRIPTION
**Pull request by lzdanski**

### CODE REVIEW

File: static/img/docs/github-integration-map.png
Comment (jwitz) [inline static/img/docs/github-integration-map.png:1]: Having "Mapped" in the description here throws me off a bit, the repo isn't mapped so much as the branches are, and their mapping is already communicated by the arrows.

Also same deal here with the boxes, they're hard to see against the black backdrop

File: static/img/docs/github-integration-deploy.png
Comment (jwitz) [inline static/img/docs/github-integration-deploy.png:1]: Really nice communication of the feature, I love how it doesn't assume any prior knowledge about CI/CD. Only feedback would be to make the boxes a little more visible, maybe with a thin white border or by making the boxes themselves lighter?

File: astro/deploy-github-integration.md [hunk 1]
```diff
@@ -12,13 +12,17 @@ Astronomer's built-in GitHub integration is the fastest way to implement CI/CD f
 
 To deploy code through an integrated GitHub repository, you first connect a GitHub repository with your Astro project to an Astro Workspace. Then, you map a Git branch in that repository to an Astro Deployment. When a pull request is merged into your mapped branch, your code is automatically deployed to Astro.
 
+<img src={require("../static/img/docs/github-integration-map.png").default} alt="A diagram showing how branches in a GitHub repository directly correlate to specific Deployments in an Astro Workspace." style={{ width: "60%", maxWidth: "400px", height: "auto" }} />
```
Comment (jwitz) [inline astro/deploy-github-integration.md:15]: Might need to play around with the dimensions in the style, currently the images are appearing v tiny. 
Screenshot 2024-04-24 at 7 38 16 AM

**Review by jwitz — CHANGES_REQUESTED** [review]
I am so excited for these to get published! Just a few comments around readability and one labeled component

File: astro/deploy-github-integration.md [hunk 2]
```diff
@@ -12,13 +12,17 @@ Astronomer's built-in GitHub integration is the fastest way to implement CI/CD f
...
+<img src={require("../static/img/docs/github-integration-map.png").default} alt="A diagram showing how branches in a GitHub repository directly correlate to specific Deployments in an Astro Workspace." />
+
 Compared to deploying code manually using the Astro CLI or through a custom CI/CD process, using Astro’s GitHub integration:
 
 - Allows you to enforce software development best practices without maintaining custom CI/CD scripts.
 - Makes it easy for developers to iterate on DAG code quickly.
 - Enables you to see Git metadata directly in the Astro UI, including Git commit descriptions.
 - Gives you greater visibility into the status and detailed logs of an individual deploy.
 
+<img src={require("../static/img/docs/github-integration-deploy.png").default} alt="A diagram showing how when an engineer commits code to a branch, then after merging the branch to GitHub, the Astro project updates." />
```
Comment (lzdanski) [inline astro/deploy-github-integration.md:24]: Tried experimenting with different size browser windows and it looks really readable at different sizes? Think I need to add the max dimensions?
Screenshot 2024-04-25 at 1 27 54 PM

Screenshot 2024-04-25 at 1 28 27 PM

**Review by jwitz — APPROVED** [review]
New sizes look GREAT! LGTM!

**Pull request merged**